plugins/org.eclipse.n4js/src/org/eclipse/n4js/external/ExternalLibraryWorkspace.java

```diff
@@ -26,6 +32,41 @@
 @ImplementedBy(NoopExternalLibraryWorkspace.class)
 public abstract class ExternalLibraryWorkspace extends InternalN4JSWorkspace {
 
+	/** Contains the projects that were built and scheduled. */
+	static public class RegisterResult {
```

Comment:
separate compilation unit?

Reply:
At the moment, only complete external projects are scheduled. Separate CUs are not considered in the external library classes